frame/lending/src/tests/vault.rs

```diff
@@ -67,3 +73,144 @@ fn test_vault_market_can_withdraw() {
 		);
 	});
 }
+
+// Struct for accounts generating
+#[derive(Encode, Decode, Default, PartialEq, Debug)]
+struct AccountPrefix([u8; 6]);
+impl TypeId for AccountPrefix {
```

Comment:
oh, not sure why it is needed, account can be anything not need for prefix, but ok.